xmlrunner/result.py

```diff
@@ -156,8 +156,10 @@ def __init__(self, test_result, test_method, outcome=SUCCESS, err=None, subTest=
 
         self.test_name = testcase_name(test_method)
         self.test_id = test_method.id()
+        self.subDescription = None
         if subTest:
             self.test_id = subTest.id()
+            self.subDescription = subTest._subDescription()
```

Comment:
should this update the `test_description` to be that of the subtest? That will allow the existing description function to work as is once sub test information is stored into the test result.

Reply:
i.e.: `self.test_result.getDescription(subTest)`

Reply:
I don't really understand what you mean... I did not find any getDescription definition in this repo?

Reply:
It's not in this repo - it's from the unittest base class which this inherits from. You can see its use a few lines above this.

Reply:
Instead of getting the description for the testcase, as we are currently doing, we can get the description for the subTest. The default behavior of `unittest` is to include the test description in the subtest.